source/develop/release-management/features/virt/engine-dedicated-cpus.md

```diff
@@ -143,5 +138,28 @@ User wants to start VM B that requires 2 vCPUs and has dedicated cpu policy
 ## Limitations
 - Should only be supported with 4.7 cluster level
 
-## TBD
-- Siblings and isolate-threads
\ No newline at end of file
+
+
+### Isolate threads/Siblings
+
+Since we already know the available host capabilities and the available CPUs
+
+in case a VM is set with Isolate threads or sibilings, we can remove the whole core from the available cpu data structure
+
+Example:
+
+Host A with 1 Socket, 2 Cores in socket, 2 CPUs in each core
```

Comment:
You can add here, that having 2 CPUs in a core indicates of a host with 2 threads topology